src/host/main.cpp

```diff
@@ -101,6 +91,17 @@ int main(int argc, char** argv)
       schema_error_msg.value()));
   }
 
+  host::CCHostConfig config = {};
+  try
+  {
+    config = nlohmann::json::parse(config_str);
```

Comment:
```suggestion
    config = config_json;
```

Avoid a double parse?